### ISSUE TITLE
Thermostat JA-150TP

### ISSUE THREAD

**Issue by DorwinDT** [kukulich/home-assistant-jablotron100#88]
Hello,
I am not able to add Jablotron thermostat JA-150TP into home assistant as card 'Thermostat'
is there some special settings for this?
I see some data from it:
![2024-02-23 14_23_14-Nastavenia – Home Assistant a 5 ďalších stránok – Osobný – Microsoft​ Edge](https://github.com/kukulich/home-assistant-jablotron100/assets/17495311/3dfdb8c8-9d16-4485-ae4b-02c7bdbafda3)

it will be nice to have such possibilities as in stock Jablotron app.

thank you for help.

**Comment by kukulich** [comment]
@DorwinDT are you able to control the thermostat in JLink or Flink application?

**Comment by DorwinDT** [comment]
@kukulich no. I am not able to control it via Flink. So it seems, that controlling of thermostat (time, temperature) is done via jablotron cloud servers? it is strange.

**Comment by kukulich** [comment]
@dfridrich are you able to modify thermostat settings via Flink?

**Comment by dfridrich** [comment]
@kukulich yes, I am. I can send you some screenshots privately.

**Comment by kukulich** [comment]
@dfridrich Přejdu do češtiny, protože se mi to bude lépe vysvětlovat.

Je třeba odchytit packety mezi tvým počítačem a Jablotronem, když budeš měnit tyto teploty. Ideálně asi přes WireShark. Pak je možné zkoušet v těch packetech dohledat změny, které si prováděl v F-linku. Tzn. ke změnám je potřeba mít čas a jakou změnu (z jaké hodnoty na jakou) si prováděl. 

Pokud se nám podaří to info v packetech dohledat, tak by mohlo být možné dodělat podporu do HA.
